docker/tutorials/docker-tutorial.md

```diff
@@ -24,6 +24,13 @@ If you've already run the command to get started with the tutorial, congratulati
 docker run -d -p 80:80 docker/getting-started
 ```
 
+> [!TIP] Make sure you are using proxy settings in Docker if necessary
+> 
+> This can be done by navigating to docker settings
+> 
+> [![](https://mermaid.ink/img/eyJjb2RlIjoiZ3JhcGggVERcbiAgICBBW0RvY2tlciBEZXNrdG9wXSAtLT5CW1NldHRpbmdzXVxuICAgIEIgLS0-IENbUmVzb3VyY2VzXVxuICAgIEMgLS0-RFtQUk9YSUVTXVxuICAgIEQgLS0-RVtUb2dnbGUgJ01hbnVhbCBwcm94eSBjb25maWd1cmF0aW9uJyB0byBvbl0iLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZ3JhcGggVERcbiAgICBBW0RvY2tlciBEZXNrdG9wXSAtLT5CW1NldHRpbmdzXVxuICAgIEIgLS0-IENbUmVzb3VyY2VzXVxuICAgIEMgLS0-RFtQUk9YSUVTXVxuICAgIEQgLS0-RVtUb2dnbGUgJ01hbnVhbCBwcm94eSBjb25maWd1cmF0aW9uJyB0byBvbl0iLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)
```

Comment:
We wouldn't want to include an image link from an exernal site, but we could download an image and check it into our repo.

Reply:
When is the proxy setting necessary?

Reply:
Good point on this. It was necessary for me internally at my company. I'm not educated enough on docker nor proxy to identify that though. Just thanking you for the feedback as I'm obviously new to this stuff and it helps me immensely understand the types of questions that are to be expected in the future and I can use.